pygmt/tests/test_meca.py

```diff
@@ -29,6 +29,25 @@ def test_meca_spec_dictionary():
     return fig
 
 
+@pytest.mark.mpl_image_compare(filename="test_meca_spec_dictionary.png")
+def test_meca_spec_dict_all_scalars():
+    """
+    Test supplying a dict with scalar values for all focal parameters.
+    """
+    fig = Figure()
+    # Right lateral strike slip focal mechanism
+    fig.meca(
+        spec=dict(
+            strike=0, dip=90, rake=0, magnitude=5, longitude=0, latitude=5, depth=0
```

Comment:
Have you tested passing in the `event_name` name as a string too? Want to see if `np.at_least1d` handles it properly.

Reply:
It works, because `np.atleast_1d("abc")` returns `array(['abc'], dtype='<U3')`.

I've revised the test to also pass a scalar `event_name` argument.